stdlib/src/sqlite.rs

```diff
@@ -1107,14 +1107,18 @@ mod _sqlite {
         ) -> PyResult<()> {
             let name = name.to_cstring(vm)?;
             let db = self.db_lock(vm)?;
-            let Some(data )= CallbackData::new(callable, vm) else {
+            let Some(data )= CallbackData::new(callable.to_owned(), vm) else {
```

Comment:
```suggestion
            let Some(data) = CallbackData::new(callable.clone(), vm) else {
```
`to_owned` came from `&PyObject`.
We are going to call `PyRc::clone(callable)` by the meaning.